lib/utils.pm

```diff
@@ -2523,7 +2523,7 @@ Leap16 will sync with Tumbleweed as well.
 =cut
 
 sub permit_root_ssh {
-    if (is_sle('<16') || is_leap('<16.0')) {
+    if (is_sle('<15-SP6') || is_leap('<16.0')) {
```

Comment:
This change could potentially break a lot of other tests if that enforcement is not seen generally in SLE15-SP6 but maybe only due to special packages pulled in that enforce that.

Reply:
yeah, I'm confused with the purpose of this if; would you mind correcting my understating please?
https://progress.opensuse.org/issues/156493#note-2

Reply:
Yeah, I also don't know why there's the condition. Consider looking into the git history for the decision.